src/Sulu/Bundle/ContentBundle/Command/SyncronizeCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Sulu\Bundle\ContentBundle\Command;
+
+use Sulu\Component\DocumentManager\DocumentHelper;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Sulu\Component\DocumentManager\DocumentManagerInterface;
+use Sulu\Component\Content\Document\SyncronizationManager;
+use Symfony\Component\Console\Input\InputOption;
+
+class SyncronizeCommand extends Command
+{
+    public function __construct(
+        DocumentManagerInterface $defaultManager,
+        SyncronizationManager $syncManager
+    )
+    {
+        parent::__construct();
+        $this->defaultManager = $defaultManager;
+        $this->syncManager = $syncManager;
+    }
+
+    public function configure()
+    {
+        $this->setName('sulu:document:syncronize');
+        $this->addOption('uuid', null, InputOption::VALUE_REQUIRED, 'UUID to syncronize');
+        $this->addOption('locale', null, InputOption::VALUE_REQUIRED, 'Locale');
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $uuid = $input->getOption('uuid');
+        $locale = $input->getOption('locale');
+
+        $document = $this->defaultManager->find($uuid, $locale);
+
+        $output->writeln(sprintf('Syncronizing document: %s', DocumentHelper::getDebugTitle($document)));
+
+        $this->syncManager->syncronizeFull($document);
```

Comment:
This is a POC command, we will likely have something similar eventually, but this is not representative of that.